Cannot build on Fedora

I get the following error on Fedora.

```
mkdir -p build
gcc -std=c11 -Wall -g -Wstrict-overflow -D_XOPEN_SOURCE=700 -D_BSD_SOURCE -I/usr/include/libpurple -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include  -I/usr/include/libxml2 -I./headers/jabber -fPIC -c ./src/carbons.c -o ./build/carbons.o
In file included from /usr/include/limits.h:25:0,
                 from /usr/lib/gcc/x86_64-redhat-linux/6.3.1/include/limits.h:168,
                 from /usr/lib/gcc/x86_64-redhat-linux/6.3.1/include/syslimits.h:7,
                 from /usr/lib/gcc/x86_64-redhat-linux/6.3.1/include/limits.h:34,
                 from /usr/lib64/glib-2.0/include/glibconfig.h:11,
                 from /usr/include/glib-2.0/glib/gtypes.h:32,
                 from /usr/include/glib-2.0/glib/galloca.h:32,
                 from /usr/include/glib-2.0/glib.h:30,
                 from ./src/carbons.c:3:
/usr/include/features.h:148:3: warning: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Wcpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"
   ^~~~~~~
gcc -fPIC -shared -std=c11 -Wall -g -Wstrict-overflow -D_XOPEN_SOURCE=700 -D_BSD_SOURCE -I/usr/include/libpurple -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include  -I/usr/include/libxml2 -I./headers/jabber ./build/carbons.o -o build/carbons.so -ldl -lm -lpurple -lglib-2.0 -lxml2 -lz -lm -ldl -L/usr/lib64/purple-2 -ljabber
/usr/bin/ld: cannot find -ljabber
collect2: error: ld returned 1 exit status
makefile:25: recipe for target 'build/carbons.so' failed
make: *** [build/carbons.so] Error 1
```

--- a/makefile
+++ b/makefile
@@ -13,7 +13,12 @@ PKGCFG_C=$(shell pkg-config --cflags glib-2.0 purple)  $(shell xml2-config --cfl
 PKGCFG_L=$(shell pkg-config --libs purple glib-2.0) $(shell xml2-config --libs) -L$(shell pkg-config --variable=plugindir purple)
 
 CFLAGS=-std=c11 -Wall -g -Wstrict-overflow -D_XOPEN_SOURCE=700 -D_BSD_SOURCE $(PKGCFG_C) $(HEADERS)
-LFLAGS= -ldl -lm $(PKGCFG_L) -ljabber
+
+ifneq ("$(wildcard /etc/redhat-release)","")
+	LFLAGS= -ldl -lm $(PKGCFG_L) -lxmpp
+else
+	LFLAGS= -ldl -lm $(PKGCFG_L) -ljabber
+endif
 
 
 all: $(BDIR)/carbons.so

